README.md

```diff
@@ -1,46 +1,63 @@
-You may also see any lint errors in the console.
+## Quick View
 
-### `npm test`
+![Untitled](https://user-images.githubusercontent.com/10102977/195940045-5685094a-0d55-4047-978d-c72625a9edef.png)
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Live Demo
 
-### `npm run build`
+[Live Demo Link](https://cosmic-choux-a8eea3.netlify.app/)
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Video Prresentation
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+[Video Presentation](https://www.loom.com/share/5449f045ce78471a9eef5abef2c1f6f9)
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Getting Started
 
-### `npm run eject`
+### Prerequisites
+- Node.js
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+### Setup
+To get a local copy up and running follow these simple example steps.
+- Clone the repo
+- cd into the project directory
+- npm install
+- npm start (to start/run the project)
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## Authors
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+👤 **Lucas Comamala**
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+- GitHub: [@lucascomamala](https://github.com/lucascomamala/)
+- LinkedIn: [Lucas Comamala](https://linkedin.com/in/lucas-comamala/)
 
-## Learn More
+## 🤝 Contributing
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+Contributions, issues, and feature requests are welcome!
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- The Microverse team
+- Riot for access to their [API](https://developer.riotgames.com/)
+- Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [OPTIONAL]  Please, let's make our readme more professional by adding a license as the link is broken. please fix this also.